test.py

```diff
@@ -0,0 +1,69 @@
+import re
+import unittest
+from typing import NoReturn
```

Comment:
not sure if using this library will bring any benefit, as all test methods always return no value

Reply:
It's commonly used technique to use type hints in such cases to improve readability

Reply:
it doesn't improve anything as test functions never return any value